frontend/src/component/changeRequest/ChangeRequestConfirmDialog/ChangeRequestConfirmDialog.tsx

```diff
@@ -1,43 +1,155 @@
+}: UpdateEnabledMsg) => (
+    <Typography>
+        <strong>{enabled ? 'Disable' : 'Enable'}</strong> feature toggle{' '}
+        <strong>{featureName}</strong> in <strong>{environment}</strong>
+    </Typography>
+);
+
+interface CopyStrategyMsg {
+    payload: IFeatureStrategy | IFeatureStrategy[];
+    fromEnvironment: string;
+    environment: string;
+}
+
+const CopyStrategy = ({
+    payload,
+    fromEnvironment,
+    environment,
+}: CopyStrategyMsg) => (
+    <Typography>
+        <strong>
+            Copy {formatStrategyName((payload as IFeatureStrategy)?.name)}{' '}
+            strategy{' '}
+        </strong>{' '}
+        from {fromEnvironment} to {environment}
+    </Typography>
+);
+
+const MsgContainer = styled('div')(({ theme }) => ({
+    '&>*:nth-child(n)': {
+        margin: theme.spacing(1, 0),
+    },
+}));
+
+const CopyStrategies = ({
+    payload,
+    fromEnvironment,
+    environment,
+}: CopyStrategyMsg) => (
+    <MsgContainer>
+        <Typography>
+            <strong>Copy: </strong>
+        </Typography>
+        {(payload as IFeatureStrategy[])?.map(strategy => (
+            <Typography>
+                <strong>
+                    {formatStrategyName((strategy as IFeatureStrategy)?.name)}{' '}
+                    strategy{' '}
+                </strong>{' '}
+            </Typography>
+        ))}
+        <Typography>
+            from {fromEnvironment} to {environment}
+        </Typography>
+    </MsgContainer>
+);
+
 export const ChangeRequestDialogue: FC<IChangeRequestDialogueProps> = ({
     isOpen,
     onConfirm,
     onClose,
+    payload,
     enabled,
     featureName,
     environment,
+    fromEnvironment,
+    variant,
+    showBanner,
 }) => (
     <Dialogue
         open={isOpen}
-        primaryButtonText="Add to draft"
+        primaryButtonText="Add suggestion to draft"
         secondaryButtonText="Cancel"
         onClick={onConfirm}
         onClose={onClose}
         title="Request changes"
+        fullWidth
     >
-        <Alert severity="info" sx={{ mb: 2 }}>
-            Change requests is enabled for {environment}. Your changes needs to
-            be approved before they will be live. All the changes you do now
-            will be added into a draft that you can submit for review.
-        </Alert>
+        {showBanner && (
+            <Alert severity="info" sx={{ mb: 2 }}>
+                Suggest changes is enabled for {environment}. Your changes needs
+                to be approved before they will be live. All the changes you do
+                now will be added into a draft that you can submit for review.
+            </Alert>
+        )}
         <Typography variant="body2" color="text.secondary">
-            Change requests:
-        </Typography>
-        <Typography>
-            <strong>{enabled ? 'Disable' : 'Enable'}</strong> feature toggle{' '}
-            <strong>{featureName}</strong> in <strong>{environment}</strong>
+            Your suggestion:
```

Comment:
Same here about reverting

Reply:
This is how the ui looks atm.

Reply:
I am not sure
![image](https://user-images.githubusercontent.com/964450/199717091-6a3df05e-6712-48e8-bcde-e70599a0787d.png)

Reply:
![Screenshot 2022-11-03 at 14 12 44](https://user-images.githubusercontent.com/104830839/199717443-756c346f-8d8f-46f0-ad78-86ce486f9151.png)

Reply:
![image](https://user-images.githubusercontent.com/964450/199741570-bf76dfbc-e7a4-4b7e-903c-35eb973bb03d.png)

Reply:
I don't get what you are pointing at.  
Before: It was `Suggested Change` - which you changed to `Change Requests`.  
Nicolae's UI after the changes specifies `Your suggestion`.
What exactly are you objecting to?